src/styles/partials/components/_subpage-tabs.scss

```diff
@@ -0,0 +1,156 @@
+.dg_subpage-tabs-container {
+  background-color: $pale-blue-dark;
+  display: flex;
+  margin-bottom: $margin-largest;
+}
+
+.dg_subpage-tabs {
+  color: $white;
+  display: block;
+  font-family: $default-heading-font;
+  font-size: $smallest-heading-font-size;
+  font-weight: $font-bold;
+  padding-left: $default-padding;
+  padding-right: $default-padding;
+  margin-right: auto;
+  margin-left: auto;
+  text-transform: uppercase;
+  width: 100%;
+
+  .dg_subpage-tabs-btn {
+    display: none;
+    h2 {
+      color: $white;
+      font-size: $small-heading-font-size;
+    }
+  }
+
+  ul {
+    list-style-type: none;
+    margin-top: $default-margin;
+    padding-left: 0px;
+
+    li {
+      float: left;
+
+      a {
+        display: block;
+        color: $white;
+        text-align: center;
+        padding: $padding-large 16px;
+        margin-right: $margin-large;
+        text-decoration: none;
+
+        &:hover {
+          color: $blue-lightest;
+        }
+      }
+
+      &.active {
+        a {
+          color: $black;
+          background-color: $white;
+        }
+      }
+    }
+  }
+}
+
+@media screen and (min-width: $extra-large-breakpoint) {
+  .dg_subpage-tabs {
+    max-width: $max-container-width;
+  }
+}
+
+@media screen and (max-width: $large-breakpoint) {
+  .dg_subpage-tabs {
+    border-top: 1px $gray solid;
```

Comment:
i believe 1px border widht is a variable

Reply:
Will take a look and update with the variable.

Reply:
Just looked at commits, didn't notice you had already updated and committed. Merged in to local branch.